src/cargo/util/errors.rs

```diff
@@ -470,7 +470,7 @@ pub fn is_simple_exit_code(code: i32) -> bool {
     // codes are very large. This is just a rough
     // approximation of which codes are "normal" and which
     // ones are abnormal termination.
-    code >= 0 && code <= 127
+    (0..=127).contains(&code)
```

Comment:
I think @ehuss has been against this one in the past.

Reply:
Ok, I reverted the change.